tex/main.tex

```diff
@@ -111,21 +113,34 @@
 
 %A self-contained abstract of about 100-200 words outlining the aims, scope and conclusions of the paper must be supplied..
  \begin{abstract}
- Ambulance services have a duty of care to the clinical outcomes of the population they serve, and therefore aim to maximise the chances of survival and improve patient outcomes following a medical emergency. Despite this, although the ambulance location-allocation problem has been widely studied, it has predominantly focused on minimising response times or maximising coverage alone, and not explicitly for considering patient outcomes. In this paper we propose a modelling approach to consider where to optimally locate different types of emergency response vehicles in order to maximise patient outcomes within a heterogeneous population.  To achieve this, we develop a heuristic algorithm for finding better fleet allocations which is used in conjunction with a discrete-event simulation model of ambulance services with heterogeneous vehicles. Our approach is informed by, and tested on, real-world data from Jakarta, Indonesia. Using our developed models, decision makers are better able to understand ambulance fleet capacity needs and locations, and their impact on patient outcomes.
+ Ambulance services have a duty of care to the clinical outcomes of the
+ population they serve, and therefore aim to maximise the chances of survival
+ and improve patient outcomes following a medical emergency. Despite this,
+ although the ambulance location-allocation problem has been widely studied,
+ it has predominantly focused on minimising response times or maximising
+ coverage alone, and not explicitly for considering patient outcomes. In this
+ paper we propose a modelling approach to consider where to optimally locate
+ different types of emergency response vehicles in order to maximise patient
+ outcomes within a heterogeneous population. To achieve this, we develop a
+ heuristic algorithm for finding better fleet allocations which is used in
+ conjunction with a discrete-event simulation model of ambulance services with
+ heterogeneous vehicles. Our approach is informed by, and tested on,
+ real-world data from Jakarta, Indonesia. Using our developed models, decision
+ makers are better able to understand ambulance fleet capacity needs and
+ locations, and their impact on patient outcomes.
```

Comment:
We should put something in the abstract about how we deal with the utilisation. This is a novel approach.